symengine/printer.cpp

```diff
@@ -425,8 +425,70 @@ void StrPrinter::bvisit(const UnivariatePolynomial &x)
 void StrPrinter::bvisit(const UnivariateSeries &x)
 {
     std::ostringstream o;
-    o << x.get_poly() << " + O(" << x.get_var() << "**" << x.get_degree()
-      << ")";
+    bool first = true;
```

Comment:
This method should go in to `UnivariateExprPolynomial`. Something like `__str__(const std::string name)` where `name` is the name of the variable.